### ISSUE TITLE
experimental effect of FECAM

### ISSUE THREAD

**Issue by AAApoint** [Zero-coder/FECAM#9]
hello, Thanks for this interesting work. I would like to ask whether the current code is the complete version?

The six data set experiments in this paper have an average increase of about 10% after adding FECAM module. But on my own dataset, I added FECAM to Autoformer and Fedformer, and the effect changed only slightly. I would like to ask what might have caused this result. 

About parameters: Is the channel value the only important parameter that can be adjusted?

**Comment by Zero-coder** [comment]
Thank you for your great question!
1.The code is current the complete version,however,we have not update the parameter setting right now for our crews are Infected with COVID-19 ,sorry for the convenience，we  need time to recover to fix this problem.
2.I would like to know more details about your dataset if it's possible.If the performance is well on autoformer and fedformer,then add FECAM would like to increase overfitting to the original model,and I would like to know where you add fecam into(eg:encoder layer)?
3.As for  the “parameter that can be adjusted”，I would like to update this work as soon as possible,sorry for the inconvenience. you can try to reduce the channel parameter in the bottleneck, 
`        self.fc = nn.Sequential(
                nn.Linear(channel, channel//num, bias=False),#channel//num
                nn.Dropout(p=0.1),
                nn.ReLU(inplace=True),
                nn.Linear( channel//num, channel, bias=False),
                nn.Sigmoid()
        )`
what's more you can implement Layernorm on different dimension (eg:(8,7,96),you can layernorm '96' or '7')
`      self.dct_norm = nn.LayerNorm(self.seq_len, eps=1e-6)#`
Thanks's for your question again!

**Comment by Zero-coder** [comment]
By the way,would you mind telling me through what channels you are aware of our work? Thank you very much.

**Comment by AAApoint** [comment]
Thank you for your reply.
I added FECAM between the FEDformer encoder and decoder as per the transformer example in your code. There was only a small improvement in prediction.
I learned about this work by searching for time series predictions on arxiv.
